mmaction/models/losses/cross_entropy_loss.py

```diff
@@ -16,13 +16,20 @@ class CrossEntropyLoss(BaseWeightedLoss):
             as None, use the same weight 1 for all classes. Only applies
             to CrossEntropyLoss and BCELossWithLogits (should not be set when
             using other losses). Default: None.
+        sparse_label (bool): Whether to use sparse labels or not. Sparse labels
```

Comment:
How about detecting which kind of loss from the shape of label?

Reply:
Sounds good. This could also simplify the codes of mixup/cutmix.